Assets/Scripts/Player/PlayerMouse.cs

```diff
@@ -0,0 +1,18 @@
+using UnityEngine;
+
+public class PlayerMouse : MonoBehaviour
```

Comment:
It doesn't have to be part of this PR, but IIRC we were gonna add health to the mech as well since there's less emphasis on getting on and off the mech.